fl4health/clients/deep_mmd_clients/ditto_deep_mmd_client.py

```diff
@@ -28,6 +28,8 @@ def __init__(
         checkpointer: Optional[ClientCheckpointModule] = None,
         deep_mmd_loss_weight: float = 10.0,
         feature_extraction_layers_with_size: Optional[Dict[str, int]] = None,
+        beta_global_update_interval: int = 20,
```

Comment:
Maybe we should call this something slightly different, since the trainable parameters in the DeepMMD loss aren't necessarily "betas"?

Reply:
We might want to scrub mention of betas throughout, since it's used in a few other variable names too.

Reply:
That is true, I should update its docstring too.